pkgs/tools/misc/whatsapp-chat-exporter/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, python3Packages, fetchFromGitHub
+
+, withEncryptedAndroid ? false
+, withCrypt12 ? false
+, withCrypt14 ? false
+, withCrypt15 ? false
+}:
+
+let version = "0.9.1";
+in python3Packages.buildPythonPackage {
```

```suggestion
in python3Packages.buildPythonApplication {
```
This is an application, not a library.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function

---

Ah, I thought I changed it. I applied your suggestion.